src/Main.java

```diff
@@ -0,0 +1,68 @@
+package src;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        //type 1
+        FirePokemon charmender = new FirePokemon(
```

Comment:
Even though you have a long list of arguments, writing it like this makes it very easily readable.